NEWS.md

```diff
@@ -26,6 +27,113 @@ The format is based on [Keep a Changelog]
 | 2021-04-15 | [0.2.0]       | 0.13.0          | 5011          |
 | 2021-03-05 | [0.1.0]       | 0.12.0          | 4627          |
 
+
+## [unreleased]
+
+### Added
+
+- Expose output configuration test to clients. For example, this enables
+  `wlr-randr --dryrun`
+- Support mapping of touch devices by name to output (display).
```

Comment:
May be also show the xml here with the `deviceName` attribute (https://github.com/labwc/labwc/blob/master/docs/rc.xml.all#L434)? Might also combine this point with the one on line 52 (“Add config option to map touch events to output names”) which introduced the `touch` element.

Reply:
Good point. Have combined + added a `deviceName` attribute  example.